### PR TITLE
Actionlog

### PR DESCRIPTION
**Pull request by jameswestman**
Adds most of the actions from #7. I haven't gotten around to testing all of them yet, but I will soon.

### CODE REVIEW

File: src/scripts/skins/original/creep.js [hunk 1]
```diff
@@ -47,18 +47,31 @@ export class CreepSkin {
...
 				default:
 					console.log("actionLog", k, a, this);
 			}
 		}
 
+		if(obj.actionLog && "say" in obj.actionLog) {
+			obj.saying = obj.actionLog.say;
+		}
+		if(obj.saying) {
+			say(this.g, obj.saying.message);
```
Comment (ricochet1k) [inline src/scripts/skins/original/creep.js]: Move this into the `case 'say'` above, and get rid of obj.saying.

File: src/scripts/actions.js [hunk 2]
```diff
@@ -47,4 +47,58 @@ export function bump(g, gRot, obj, a) {
...
+		}
+
+		if (v < 0.5) {
+			v = v;
+		} else {
+			v = (1 - v);
+		}
+
+		g.clear();
+		g.beginFill(color, v);
```
Comment (ricochet1k) [inline src/scripts/actions.js]: It's probably much better to set g.alpha instead of redrawing every frame

File: src/scripts/actions.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+import {SQUARE_SIZE, TWEEN_DURATION, S} from './const';
 import {tween, tweenRotation, interp} from './tween';
 
-export function actionLine(room, action, from, to) {
+export function actionLine(room, action, from, to, color) {
 	let g = new PIXI.Graphics();
 	room.g.addChild(g);
 
-	const color = 0xffff00;
+	color = color || 0xffff00;
```
Comment (ricochet1k) [inline src/scripts/actions.js]: This should be added as a default value to the color parameter above, like this: `, color = 0xffff00) {`

**Review by ricochet1k — CHANGES_REQUESTED** [review]
Looks pretty good, fix a few things and I'll merge this. Sorry it took me so long to review.

**Comment by jameswestman** [comment]
I've made the changes you suggested. (I'm new to PixiJS, thanks for catching my mistakes!)

**Review by ricochet1k — APPROVED** [review]

**Pull request merged**